### PR TITLE
Fix libprotobuf-lite build

### PR DESCRIPTION
**Pull request by N00byEdge**
Fixes the absolute paths to @heinermann 's download folder and uses the relative path. Also removed the unused imports from projects not included.

### CODE REVIEW

File: bwapi/Backend/libprotobuf-lite/libprotobuf-lite.vcxproj [hunk 1]
```diff
@@ -103,53 +103,53 @@
...
+    <ClInclude Include="src\google\protobuf\stubs\statusor.h" />
+    <ClInclude Include="src\google\protobuf\stubs\stringpiece.h" />
+    <ClInclude Include="src\google\protobuf\stubs\stringprintf.h" />
+    <ClInclude Include="src\google\protobuf\stubs\strutil.h" />
+    <ClInclude Include="src\google\protobuf\stubs\time.h" />
+    <ClInclude Include="src\google\protobuf\wire_format_lite.h" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="C:\Users\Adam Heinermann\Downloads\protobuf-cpp-3.6.0\protobuf-3.6.0\install\ZERO_CHECK.vcxproj">
+    <ProjectReference Include="install\ZERO_CHECK.vcxproj">
```
Comment (heinermann) [inline bwapi/Backend/libprotobuf-lite/libprotobuf-lite.vcxproj]: can probably get rid of this guy, no idea what it is

Comment (N00byEdge) [reply bwapi/Backend/libprotobuf-lite/libprotobuf-lite.vcxproj]: Yes we should remove that first I guess

**Comment by N00byEdge** [comment]
Also may I propose adding the `.appveyor.yml` to the repo? That way, this branch can have its own AppVeyor configuration and actually build.

image

**Comment by heinermann** [comment]
Yeah sounds good

**Pull request merged**